_posts/0000-01-02-J-mot3000.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![image](https://user-images.githubusercontent.com/70128193/95274441-7a08ed80-080b-11eb-9729-6a9493b873fc.png)

Reply:
How about this meme?